src/location/qgeomap.cpp

```diff
@@ -370,7 +370,7 @@ void QGeoMapMapLibrePrivate::threadedRenderingHack(QQuickWindow *window, Map *ma
     // in MapLibre Native. Meanwhile we need to set a timer to update
     // the map until all the resources are loaded, which is not exactly
     // battery friendly, because might trigger more paints than we need.
-    if (!m_warned) {
+    if (!m_threadedRenderingChecked) {
```

Comment:
```suggestion
    if (!m_threadedRendering) {
```

Reply:
Hmm, this would make it check every time if threaded rendering is disabled. Not sure we want that.